atmo/jobs/models.py

```diff
@@ -406,17 +407,54 @@ def sync(self, info=None):
             ('ready_datetime', 'ready_at'),
             ('end_datetime', 'finished_at'),
         )
+        save_needed = False
+        date_fields_updated = False
+
         # set the various model fields to the value the API returned
         for api_field, model_field in model_field_map:
             field_value = info.get(api_field)
-            if field_value is None:
+            if field_value is None or field_value == getattr(self, model_field):
                 continue
             setattr(self, model_field, field_value)
+            save_needed = True
```

Comment:
This is basically a rewrite of the ["update_fields" feature](https://docs.djangoproject.com/en/1.11/ref/models/instances/#specifying-which-fields-to-save) in Django, let's use this instead, no need to track the state ourselves.

Reply:
What I'm trying to do is avoid a `save()` if the data didn't change. I noticed that calling `save()` on every sync call made the front-end trigger a UI refresh notice when no actual change actually occurs.

Does using `update_fields` bypass the auto-now stuff for the modified column?